### PR TITLE
Gives Dionae Nral'Malic and Sol Common, as well as an additional language slot

### PR DESCRIPTION
**Pull request by Geevies**
* Nral
* Malic
* Sol
* Common
* Language
* Slot

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by SleepyWolfe** [comment]
[crab dance head lore dev said yes](https://i.gyazo.com/c5ea5bb1bb402bd41025da27a9e1f074.png)
please add Sinta'Unathi as well @Geevies Marlon said yes B)

File: code/modules/mob/living/carbon/human/species/station/diona/diona.dm [hunk 1]
```diff
@@ -23,7 +23,8 @@
 	siemens_coefficient = 0.3
 	eyes = "blank_eyes"
 	show_ssd = "completely quiescent"
-	num_alternate_languages = 1
+	num_alternate_languages = 2
+	secondary_langs = list(LANGUAGE_SOL_COMMON, LANGUAGE_SKRELLIAN)
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/species/station/diona/diona.dm]:
```suggestion
	secondary_langs = list(LANGUAGE_SOL_COMMON, LANGUAGE_SKRELLIAN, LANGUAGE_AZAZIBA)
```

File: html/changelogs/geeves-iAmTheLorax.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Dionae can now speak Sol Common and Nral'Malic."
```
Comment (Alberyk) [inline html/changelogs/geeves-iAmTheLorax.yml]: It is missing azaziba here,

Comment (Geevies) [inline html/changelogs/geeves-iAmTheLorax.yml]:
```suggestion
  - rscadd: "Dionae can now speak Sol Common, Sinta'Azaziba and Nral'Malic."
```

**Comment by Arrow768** [comment]
Missing pre-reqs. DNM until they are fulfilled.

**Pull request closed without merging**